CSharp/BatchExplorer/Service/BatchService.cs

```diff
@@ -201,13 +201,48 @@ public Task<CloudPool> GetPoolAsync(string poolId)
             int targetDedicated,
             TimeSpan? timeout,
             ComputeNodeDeallocationOption? deallocationOption)
-        {
+        {
+            var pool = await this.GetPoolAsync(poolId);
+            if (pool.AutoScaleEnabled.HasValue && pool.AutoScaleEnabled.Value)
+            {
+                await this.Client.PoolOperations.DisableAutoScaleAsync(poolId);
+                if (!await this.WaitForResizingOperationToFinish(poolId))
+                {
+                    throw new InvalidOperationException("Autoscale disable operation is already in progress. Please try again after sometime to Resize the pool.");

```

Comment:
@aforank - So it sounds like we're saying we think removing this check + wait is best? Instead just perform the resize and if they hit this error they will get a popup which they can do with as they will (either retry in 10s, or whatever)

@paselem - Is that what we're suggesting?